### ISSUE TITLE
[P1] iPEPS excitation normalization removes the quadratic dependence on the excitation tensor

### ISSUE THREAD

**Issue by yingjerkao** [tenax-lab/tenax#954]
> 🤖 **AI-generated issue** - written by Codex from the requested iPEPS code review.

The mixed-RDM helpers normalize each B-dependent contraction by its own trace before the excitation energy and norm are differentiated. Scaling B is then canceled by the denominator. This violates the quadratic-form assumption used to construct H_eff and N from gradients at basis vectors, so the reported generalized eigenproblem is not the advertised excitation problem.

Sources: [horizontal normalization](https://github.com/tenax-lab/tenax/blob/42aef1f1a94e1ca8400f6187400aa3613e3e175e/src/tenax/algorithms/ipeps_excitations.py#L249-L251), [vertical normalization](https://github.com/tenax-lab/tenax/blob/42aef1f1a94e1ca8400f6187400aa3613e3e175e/src/tenax/algorithms/ipeps_excitations.py#L284-L286), [matrix-construction assumption](https://github.com/tenax-lab/tenax/blob/42aef1f1a94e1ca8400f6187400aa3613e3e175e/src/tenax/algorithms/ipeps_excitations.py#L537-L548).

## Reproduction

An exact D=chi=1 product environment excludes finite-chi convergence errors:

```python
import jax.numpy as jnp
from tenax.algorithms.ipeps_config import CTMEnvironment
import tenax.algorithms.ipeps_excitations as ex

env = CTMEnvironment(
    *([jnp.ones((1, 1))] * 4 + [jnp.ones((1, 1, 1))] * 4)
)
A = jnp.array([1., 2.]).reshape(1, 1, 1, 1, 2) / jnp.sqrt(5.)
B = jnp.array([3., 4.]).reshape(A.shape)
k = jnp.array([0.4, 0.7])
H = jnp.diag(jnp.array([0., 1., 1., 2.])).reshape(2, 2, 2, 2)
for scale in (1., 2.):
    print(scale,
          ex._compute_norm(A, scale * B, env, k, 2),
          ex._compute_excitation_energy(A, scale * B, env, k, H, 0., 2))
```

Actual output:

```text
1.0 7.5040377675334256 10.722627436472361
2.0 7.5040377675334256 10.722627436472361
```

Both quantities should grow by a factor of four when B doubles. The problem is independent of the separate physical-axis-ordering defect in these helpers: B-dependent division would still cancel amplitude after correcting the axes.

## Suggested Fix

Normalize transition contractions against a common B-independent ground-state contraction. Preserve the unsymmetrized mixed transition operators until conjugate/momentum contributions are combined; individually Hermitian-symmetrizing each transition operator is also inappropriate. Add scaling tests for real and complex multipliers and quadratic reconstruction tests for H_eff and N.

Reproduced on commit `42aef1f1a94e1ca8400f6187400aa3613e3e175e` using `JAX_PLATFORMS=cpu JAX_ENABLE_X64=1 uv run python`. No repository code was modified for these probes.
